tests/components/light/test_trigger.py

```diff
@@ -106,8 +116,30 @@ async def target_lights(hass: HomeAssistant) -> None:
 @pytest.mark.parametrize(
     ("trigger", "initial_state", "states"),
     [
+        ("light.turned_on", None, [(STATE_ON, 1), (STATE_OFF, 0), (STATE_ON, 1)]),
+        ("light.turned_off", None, [(STATE_OFF, 1), (STATE_ON, 0), (STATE_OFF, 1)]),
```

Comment:
Can we create constants for the two different state sequences?
Each test uses:
- ON_OFF_ON:  `[(STATE_ON, 1), (STATE_OFF, 0), (STATE_ON, 1)]`
- OFF_ON_OFF:  `[(STATE_OFF, 1), (STATE_ON, 0), (STATE_OFF, 1)]`